packages/@okta/vuepress-site/.vuepress/nav/codePages.js

```diff
@@ -12,9 +12,60 @@ module.exports = [
   {
     title: "Front End",
     subLinks: [
-      { title: 'Angular', path: '/code/angular/' },
-      { title: 'JavaScript', path: '/code/javascript/' },
-      { title: 'React', path: '/code/react/' },
+      { 
+        title: 'Angular',
+        subLinks: 
+        [
+          {
+            title: 'Add User Authentication to Your Angular App',
+            path: '/code/angular/',
+          },
+          {
+            title: 'Angular Sign in widget',
```

Comment:
content question: should this be aligned with other `Sign-In Widget` mentions?

Reply:
I think it should. 2 of 3 are Sign-In Widget. Fixed.